client/src/api/socketHandler/socketListeners.js

```diff
@@ -25,4 +25,26 @@ const newClientJoined = () => {
   socket.on('newClientJoinedUserRoom', (message) => message);
 };
 
-export { successRoomConnection, errorRoomConnection, newClientJoined };
+/**
+ * @description Register listener on when a new robot log update has been send
+ */
+const newRobotMonitorUpdate = (logSetterMethod) => {
+  socket.on('liveRobotMonitoring', (robotLogs) => {
```

Comment:
Not sure if this is an appropriate name for the event. All other event types seem to describe something that happened while this name seems to be off from that